routes/projects/projects-router.js

```diff
@@ -1,29 +1,77 @@
-const express = require('express')
+const express = require("express");
+
+const router = express.Router();
+const Projects = require("./projects-model");
 
-const router = express.Router()
-const projects = require('./projects-model')
- 
 //endpoint to get all projects
-router.get('/', (req, res) => {
-    projects.getProjects()
-    .then(projects => {
-        res.status(200).json(projects)
-    })
-    .catch(err => {
-        console.log(err)
-        res.status(500).json(err)
-    })
-})
+router.get("/", async (req, res) => {
+  try {
+    const projects = await Projects.getProjects();
+    res.status(200).json(projects);
+  } catch (error) {
+    console.log("Getting projects error: ", error);
+    res.status(500).json({ message: "error getting projects ", error });
+  }
+});
+
```

Comment:
Great job on the error handling messages on these! Can't remember the amount of times the BE guys on my projects just skipped over those 🔥